docs/integrations/storage-connectors/snowflake.md

```diff
@@ -0,0 +1,48 @@
+Snowflake is a popular managed data warehouse on AWS, Azure, and GCP.
+
+<!--
+<p align="center">
+  <figure>
+    <img src="../../../assets/images/storage-connectors/snowflake.png" alt="Setup a Snowflake storage connector">
+    <figcaption>Configure the Snowflake storage connector in the Hopsworks UI.</figcaption>    
+  </figure>
+</p>
+-->
+
+In the UI for the Snowflake connector, you should enter the following:
+
+The following options are required to create a snowflake connector:
+- url: the hostname for your account in the following format: <account_name>.snowflakecomputing.com.
+- user: login name for the Snowflake user.
+- password: password of the Snowflake user. (required if token is not set)
+- token: OAuth token that can be used to access snowflake. (required if password is not set)
+- database: the database to use for the session after connecting.
+- schema: the schema to use for the session after connecting.
+
+The remaining options are not required, but are optional:
+- warehouse: the default virtual warehouse to use for the session after connecting.
+- role: the default security role to use for the session after connecting.
+- table: the table to which data is written to or read from. 
+
+Additional snowflake options can be added as a list of key-value pair in sfOptions
+
+There are two options available for authentication. The first option is to configure a username and a password. The second option is to use an OAuth token. See [Configure Snowflake OAuth](https://docs.snowflake.com/en/user-guide/oauth-custom.html) for instruction on how to configure OAuth support for snowflake, and [Using External OAuth](https://docs.snowflake.com/en/user-guide/spark-connector-use.html#using-external-oauth) on how you can use External OAuth to authenticate to Snowflake.
+
+With regards to the database driver, the library to interact with Snowflake *is not* included in Hopsworks - you need to upload the driver yourself. First, you need to [download the jdbc driver](https://repo1.maven.org/maven2/net/snowflake/snowflake-jdbc) and to use snowflake as the data source in spark [the snowflake spark connector](https://docs.snowflake.com/en/user-guide/spark-connector-install.html).
```

Comment:
This page is not properly formatted. something is wrong with the bullet points.

![image](https://user-images.githubusercontent.com/8422705/109785573-8d7a8200-7c0c-11eb-8f56-391706fe8e27.png)

Reply:
I fixed the bullet points. When i run 'mkdocs serve', it looks ok.